src/providers/bttv/BttvEmotes.hpp

```diff
@@ -33,6 +35,41 @@ class BttvEmotes final
                             std::function<void(EmoteMap &&)> callback,
                             bool manualRefresh);
 
+    /**
+     * Adds an emote to the `map`.
+     * This will _copy_ the emote map and
+     * update the `Atomic`.
+     *
+     * @return The added emote.
+     */
+    static EmotePtr addEmote(
+        const QString &channelDisplayName,
+        Atomic<std::shared_ptr<const EmoteMap>> &map,
+        const BttvLiveUpdateEmoteUpdateAddMessage &message);
```

Comment:
Can the `map` parameter and/or comments be updated to clarify that this function modifies channel specific emotes?